### PR TITLE
UT: scraper should run, added related bills to events

### PR DESCRIPTION
**Pull request by rshorey**
Related bills was an easy lift while I was in there, @paultag assume this is something we want?

### CODE REVIEW

File: openstates/ut/events.py [hunk 1]
```diff
@@ -14,7 +14,9 @@ def scrape(self, session, chambers):
         events = doc.xpath('//item')
 
         for info in events:
-            (title, when) = info.xpath('title/text()')[0].split(" - ")
+            title_and_date = info.xpath('title/text()')[0].split(" - ")
+            title = title_and_date[0]
+            when = title_and_date[-1]
```
Comment (paultag) [inline openstates/ut/events.py]: Out of curiosity, what exists twixt these two?

File: openstates/ut/events.py [hunk 2]
```diff
@@ -36,18 +38,38 @@ def scrape(self, session, chambers):
...
+                url = url.group(1)
                 event.add_document(
                         name=document.xpath('text()')[0],
-                        url=re.search(r'(http://.*?pdf)', document.xpath('@onclick')[0]).group(1),
+                        url=url,
                         mimetype='application/pdf'
                         )
+                bills = document.xpath('@onclick')
+                for bill in bills:
+                    if "2015/bills/static" in bill:
```
Comment (paultag) [inline openstates/ut/events.py]: Hurm; can we tweak this to just `/bills/static`, or does that capture too much?

Comment (paultag) [inline openstates/ut/events.py]: (just for future upgrade sanity)

File: openstates/ut/events.py (hunk 1 above)
Comment (rshorey) [inline openstates/ut/events.py]: I think it was a time

Comment (rshorey) [inline openstates/ut/events.py]: but it was not consistent, just once

Comment (rshorey) [inline openstates/ut/events.py]: fixed

**Comment by paultag** [comment]
:+1: Thanks!

**Pull request merged**